frontend/packages/ux-editor/src/containers/DesignView/FormLayout.tsx

```diff
@@ -4,17 +4,22 @@ import React from 'react';
 import { hasMultiPageGroup } from '../../utils/formLayoutUtils';
 import { useTranslation } from 'react-i18next';
 import { Alert, Paragraph } from '@digdir/design-system-react';
+import { FormLayoutWarning } from './FormLayoutWarning';
 
 export interface FormLayoutProps {
   layout: IInternalLayout;
+  isValid: boolean;
 }
 
-export const FormLayout = ({ layout }: FormLayoutProps) => (
-  <>
-    {hasMultiPageGroup(layout) && <MultiPageWarning />}
-    <FormTree layout={layout} />
-  </>
-);
+export const FormLayout = ({ layout, isValid }: FormLayoutProps) =>
```

Comment:
What about writing it like this? 😄 

```suggestion
export const FormLayout = ({ layout, isValid }: FormLayoutProps) => {
   if (!isValid) {
      return (
         <FormLayoutWarning layout={layout} />
      )
   }
   return (
      <>
         {hasMultiPageGroup(layout) && <MultiPageWarning />}
         <FormTree layout={layout} />
     </>
   )
}
```